src/device/tun_linux.rs

```diff
@@ -45,6 +45,9 @@ pub struct ifreq {
     ifr_ifru: IfrIfru,
 }
 
+#[cfg(target_env = "musl")]
+const SIOCGIFMTU: int32_t = 0x8921;
```

Comment:
I think the best way to handle it is to cast using 'as _' wherever SIOCGIFMTU is used.